web/backend/src/cli.ts

```diff
@@ -65,4 +67,36 @@ program
     console.log('Permissions removed successfully!');
   });
 
+// Imports a list of SCIPERS from a file to allow to vote on a specific election
+// the .voters.example file is available as an example
+program
+  .command('addVoters')
+  .description('Assigns a list of SCIPERs to an Election as Voters')
+  .requiredOption('-e, --election-id <char>', 'ID of the election')
+  .requiredOption('-sf, --scipers-file <char>', 'File with line-separated list of SCIPERs')
+  .action(async ({ electionId, scipersFile }) => {
+    fs.readFile(scipersFile, 'utf8', async (err: any, data: string) => {
+      if (err) {
+        throw new InvalidArgumentError(`Faced a problem trying to process your file: \n ${err}`);
+      }
+      const scipers: Array<string> = data.split('\n');
+      const policies = [];
+      for (let i = 0; i < scipers.length; i += 1) {
+        const sciper: number = Number(scipers[i]);
+        if (Number.isNaN(sciper)) {
+          throw new InvalidArgumentError(`SCIPER on line ${i + 1} is not a number - exiting!`);
+        }
+        if (sciper > 999999 || sciper < 100000) {
+          throw new InvalidArgumentError(
+            `SCIPER on line ${i + 1} is outside acceptable range (100000..999999) - exiting!`
```

Comment:
Having the line number is nice, but I'd consider outputting the SCIPER itself within quotes, so as to remove any ambiguity.

E.g.

```suggestion
          throw new InvalidArgumentError(`SCIPER '${sciper}' on line ${i + 1} is not a number - exiting!`);
        }
        if (sciper > 999999 || sciper < 100000) {
          throw new InvalidArgumentError(
            `SCIPER '${sciper}' on line ${i + 1} is outside acceptable range (100000..999999) - exiting!`
```